hollow/src/main/java/com/netflix/hollow/api/producer/HollowIncrementalProducer.java

```diff
@@ -86,10 +121,10 @@ private void removeRecords(WriteState newState) {
 
             private Map<String, BitSet> findTypesWithRemovedRecords(ReadState readState) {
                 Map<String, BitSet> recordsToRemove = new HashMap<String, BitSet>();
-                for(RecordPrimaryKey key : mutations.keySet()) {
-                    if(!recordsToRemove.containsKey(key.getType())) {
+                for (RecordPrimaryKey key : mutations.keySet()) {
```

Comment:
Though unconventional, we prefer omitting the space between `for`, `if`, `while`, etc. and the following open paren.